### ISSUE TITLE
Datmo init fails in empty directory (and ssh git credentials not present) for Windows10

### ISSUE THREAD

**Issue by nmwalsh** [datmo/datmo#141]
Example:
```
PS C:\Users\dev\Projects> cd example_datmo_project
PS C:\Users\dev\Projects\example_datmo_project> datmo init
Initializing project None @ (C:\Users\dev\Projects\example_datmo_project)
Enter name for the project: example datmo project
Enter description for the project: test
The authenticity of host 'github.com (192.30.255.113)' can't be established.
RSA key fingerprint is <RSA KEY>.
Are you sure you want to continue connecting (yes/no)? yes
Failed to create project example datmo project @ (C:\Users\dev\Projects\example_datmo_project)
```

Datmo init fails in Windows10 64-bit for empty repository

**Comment by asampat3090** [comment]
@nmwalsh  Does this still fail once you have setup global git credentials? It looks like we will have to add tests for the case where git credentials are not given, tests for empty repositories are already present, but do not test if git credentials are not given, which seems to be the error

**Comment by nmwalsh** [comment]
Set up global git credentials (Uses 2FA, stored token in Windows Git Credentials Manager) and still fails. Both before and after, a .datmo folder is still created with the following:

![datmo-init](https://user-images.githubusercontent.com/10050819/40261350-f86493de-5ab4-11e8-8c37-3656134b8c8c.png)

**Comment by asampat3090** [comment]
The issue of the `.datmo` folder is a separate issue, so there are 2 commits to fix this. For the first commit 69234d6 and for the second 6441a40

**Comment by asampat3090** [comment]
closed with PR #144
